src/index.ts

```diff
@@ -39,15 +39,11 @@ import {
 const getDefaultInitParams = (initParams?: InitParams) => {
   const p: InitParams = {
     network: 'testnet',
```

Comment:
Question: Why are we removing the inpage provider field from the initial params?

Reply:
Because `inpageProvider` was mainly for `window.ethereum` but it is unreliable so we are not using it.
`window.arcana.provider` will always be set on `init()` so don't need to have that option now.

Reply:
+ @karthik-durai fyi, we'll need to make this change in the private nft minter.

```
  storage = new StorageProvider({
    appId: APP_ID,
    gateway: GATEWAY_URL,
    chainId: Number(BLOCKCHAIN_ID),
    provider: window.arcana.provider, // <- change from window.ethereum
    debug: true,
  })
```